### ISSUE TITLE
Kustomize.io based Kubernetes Manifests

### ISSUE THREAD

**Issue by davidasnider** [Optum/barista#85]
I think a Kustomize based set of Kubernetes manifests for dev, and production style environments would help with adoption.  If you agree, I'd be happy to submit the PR.

**Comment by pkn4645** [comment]
@rolinge @woodrbe can you weigh in?

**Comment by woodrbe** [comment]
Looks interesting.  @davidasnider, will take any help that makes the product better.

**Comment by pkn4645** [comment]
> I think a Kustomize based set of Kubernetes manifests for dev, and production style environments would help with adoption. If you agree, I'd be happy to submit the PR.

@davidasnider would this assist with enabling a simplified demo environment for new community users?  Something that could more quickly be up and running locally so people interested in barista could evaluate functionality and provide feedback?

**Comment by davidasnider** [comment]
absolutely.  I'm working through skaffold right now to duplicate some of the Docker compose build functionality.  It would be WAAAY easier if the containers were built and published on a public repository.

**Comment by pkn4645** [comment]
@rolinge given @davidasnider working on this issue, would now be a good time to publish our containers on a public repo?  I know we had issues getting control of the Optum namespace on docker hub, but what about something like canister.io as an alternative?

**Comment by rolinge** [comment]
> @rolinge @woodrbe can you weigh in?
Any place is fine, does not have to be docker hub.  Quay.io is Redhat's repo, that is quite popular too.  I'm not sure if it has a free tier, though.  Wherever we decide to put the built images, we just put those is the docker-compose file.

Regarding kustomize, not trying to start a land war in Asia or anything, but I already wrote up the helm templates for Kubernetes.  The problem with those (or Kustomize) is that they are very specific to the environment.  We have all the custom settings pulled out into variables, so it should be something we can publish.  It is just not as easy as doing a compose file.
